orion/primitives/aer.py

```diff
@@ -0,0 +1,316 @@
+        fy_hat (ndarray):
+            Predicted values (forward).
+        smoothing_window (float):
+            Optional. Size of the smoothing window, expressed as a proportion of the total
+            length of y. If not given, 0.01 is used.
+        smooth (bool):
+            Optional. Indicates whether the returned errors should be smoothed with EWMA.
+            If not given, `True` is used.
+        mask (bool): bool = True
+            Optional. Mask anomaly score errors in the beginning.
+            If not given, `True` is used.
+    Returns:
+        ndarray:
+            Array of errors.
+    """
+    time_steps = len(y[0]) - 1
+    mask_steps = int(smoothing_window * len(y)) if mask else 0
+    ry, fy = y[:, 0], y[:, -1]
+
+    f_scores = regression_errors(fy, fy_hat, smoothing_window=smoothing_window, smooth=smooth)
+    f_scores[:mask_steps] = 0
+    f_scores = np.concatenate([np.zeros(time_steps), f_scores])
+
+    r_scores = regression_errors(ry, ry_hat, smoothing_window=smoothing_window, smooth=smooth)
+    r_scores[:mask_steps] = min(r_scores)
+    r_scores = np.concatenate([r_scores, np.zeros(time_steps)])
+
+    scores = f_scores + r_scores
+    scores[time_steps + mask_steps:-time_steps] /= 2
+    return scores
+
+
+def score_anomalies(y: ndarray, ry_hat: ndarray, y_hat: ndarray, fy_hat: ndarray,
+                    smoothing_window: float = 0.01, smooth: bool = True, mask: bool = True,
+                    comb: str = 'mult', lambda_rec: float = 0.5, rec_error_type: str = "dtw"):
+    """Compute an array of absolute errors comparing predictions and expected output.
+
+    If smooth is True, apply EWMA to the resulting array of errors.
+
+    Args:
+        y (ndarray):
+            Ground truth.
+        ry_hat (ndarray):
+            Predicted values (reverse).
+        y_hat (ndarray):
+            Predicted values. Each timestamp has multiple predictions.
+        fy_hat (ndarray):
+            Predicted values (forward).
+        smoothing_window (float):
+            Optional. Size of the smoothing window, expressed as a proportion of the total
+            length of y. If not given, 0.01 is used.
+        smooth (bool):
+            Optional. Indicates whether the returned errors should be smoothed with EWMA.
+            If not given, `True` is used.
+        mask (bool): bool = True
+            Optional. Mask anomaly score errors in the beginning.
+            If not given, `True` is used.
+        rec_error_type (str):
+            Optional. The method to compute reconstruction error. Can be one of
+            `["point", "area", "dtw"]`. If not given, 'dtw' is used.
+        comb (str):
+            Optional. How to combine critic and reconstruction error. Can be one
+            of `["mult", "sum", "rec"]`. If not given, 'mult' is used.
+        lambda_rec (float):
+            Optional. Used if `comb="sum"` as a lambda weighted sum to combine
+            scores. If not given, 0.5 is used.
+    Returns:
```

Comment:
minor: add space between the last a item in argument and returns.

Reply:
Added